ocaml/tests/common/test_common.ml

```diff
@@ -15,10 +15,10 @@
 open API
 
 (* A directory to use for temporary files. *)
-let working_area = "/tmp/xapi-test"
-
-(** Utility functions *)
-let id (x : 'a) : 'a = x
+let working_area =
+  Filename.concat
+    (Option.value ~default:"/tmp" (Sys.getenv_opt "TMPDIR"))
```

Comment:
Doesn't Filename module in OCaml have a way to read/use TMPDIR automatically? See `Filename.get_temp_dir_name`

Reply:
We could also use Filename.temp_file which also takes care of creating a unique temp name with a chosen suffix (although cleaning it up is up to the caller).